frontend/apps/reader/modules/readergesture.lua

```diff
@@ -783,7 +783,11 @@ function ReaderGesture:gestureAction(action, ges)
             })
         end
     elseif action == "suspend" then
-        UIManager:suspend()
+        if Device:isKindle() then
```

Comment:
I assume this predates your other fix? The changes to this file are redundant and wouldn't belong here even if they weren't. ;-)

Reply:
Oh yes, I accidentally added this, I'll remove it in a moment